src/main/resources/category/java/common_std.xml

```diff
@@ -39,4 +39,50 @@
                 ]]>
         </example>
     </rule>
+
+    <rule name="AvoidConcatInAppend"
+          language="java"
+          message="Concatenation inside append. Use extra append."
+          class="net.sourceforge.pmd.lang.rule.xpath.XPathRule"
+          typeResolution="true"
+          externalInfoUrl="${doc_root}/JavaCodePerformance.md#isu04">
+        <description>Concatenation of Strings is used inside an StringBuilder.append argument. Problem: Each statement with one or more +-operators creates a hidden temporary StringBuilder, a char[] and a new String object, which all have to be garbage collected.&#13;
+            Solution: Use an extra fluent append instead of concatenation.
+        </description>
+        <priority>2</priority>
+        <properties>
+            <property name="tags" value="jpinpoint-rule,performance" type="String" description="classification"/>
+            <property name="xpath">
+                <value><![CDATA[
+//(ReturnStatement|ExpressionStatement|LocalVariableDeclaration)
+[.//MethodCall[@MethodName="append"]
```

Comment:
Ik zou hier matchesSig gebruiken om zeker te weten dat het over StringBuilder gaat 
(of StringBuffer, heb ik wel gedaan in een regel, maar we hebben ook een regel dat je geen StringBuffer moet gebruiken dus wellicht niet nodig)

Reply:
changed to matchesSig and added StringBuffer.append